tools/nbfmt.py

```diff
@@ -128,17 +128,17 @@ def clean_cells(data) -> None:
     cell_meta.pop("executionInfo", None)
     cell["metadata"] = cell_meta
 
-    # Add `outputs` field to code cells if it doesn't exist since its a required
-    # property.
-    if cell.get("outputs", None) is None:
-      cell["outputs"] = []
-
-    if private_outputs:
+    # Clear any code cell outputs if notebook set to private outputs.
+    if private_outputs and cell.get("outputs"):
       has_outputs = True
       # Clear code outputs
       cell["execution_count"] = 0
       cell["outputs"] = []
 
+    # Ensure outputs field exists since part of the nbformat spec.
+    if cell.get("outputs", None) is None:
```

Comment:
Why move it?

Reply:
because it's triggering a warning it shouldn't (from chat)